.github/workflows/deploy.yaml

```diff
@@ -139,3 +139,5 @@ jobs:
         with:
           user: __token__
           password: ${{ secrets.pypi_password }}
+          skip_existing: true
+
```

Comment:
I don't know what this does, so I'll trust you that it's useful and intended :)

Reply:
It is intended, but not a huge problem in pyresample given the way our CI is set up. I noticed it while working on another pytroll package. Basically, if PyPI has a specific package already and you try to upload it again the CI job would fail. In any case where this would be a problem we would likely be uploading a new version of a wheel (ex. new version of Python without a new pyresample release, just new wheel build for that version). We don't want the upload to fail just because there are existing wheels/files.